### ISSUE TITLE
detection not working

### ISSUE THREAD

**Issue by ogulcankarayel5** [AlexeyAB/darknet#4584]
When I start the detection like below just same image is openning.
```
> C:\Users\karay\darknet>darknet.exe detector test cfg/plate-obj.data cfg/plate-yolov3-tiny_obj.cfg backup\plate-yolov3-tiny_obj_last.weights data/test.jpg
> net.optimized_memory = 0
> batch = 1, time_steps = 1, train = 0
>    layer   filters  size/strd(dil)      input                output
>    0 conv     16       3 x 3/ 1    416 x 416 x   3 ->  416 x 416 x  16 0.150 BF
>    1 max                2x 2/ 2    416 x 416 x  16 ->  208 x 208 x  16 0.003 BF
>    2 conv     32       3 x 3/ 1    208 x 208 x  16 ->  208 x 208 x  32 0.399 BF
>    3 max                2x 2/ 2    208 x 208 x  32 ->  104 x 104 x  32 0.001 BF
>    4 conv     64       3 x 3/ 1    104 x 104 x  32 ->  104 x 104 x  64 0.399 BF
>    5 max                2x 2/ 2    104 x 104 x  64 ->   52 x  52 x  64 0.001 BF
>    6 conv    128       3 x 3/ 1     52 x  52 x  64 ->   52 x  52 x 128 0.399 BF
>    7 max                2x 2/ 2     52 x  52 x 128 ->   26 x  26 x 128 0.000 BF
>    8 conv    256       3 x 3/ 1     26 x  26 x 128 ->   26 x  26 x 256 0.399 BF
>    9 max                2x 2/ 2     26 x  26 x 256 ->   13 x  13 x 256 0.000 BF
>   10 conv    512       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 512 0.399 BF
>   11 max                2x 2/ 1     13 x  13 x 512 ->   13 x  13 x 512 0.000 BF
>   12 conv   1024       3 x 3/ 1     13 x  13 x 512 ->   13 x  13 x1024 1.595 BF
>   13 conv    256       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x 256 0.089 BF
>   14 conv    512       3 x 3/ 1     13 x  13 x 256 ->   13 x  13 x 512 0.399 BF
>   15 conv     18       1 x 1/ 1     13 x  13 x 512 ->   13 x  13 x  18 0.003 BF
>   16 yolo
> [yolo] params: iou loss: mse (2), iou_norm: 0.75, cls_norm: 1.00, scale_x_y: 1.00
>   17 route  13                                     ->   13 x  13 x 256
>   18 conv    128       1 x 1/ 1     13 x  13 x 256 ->   13 x  13 x 128 0.011 BF
>   19 upsample                 2x    13 x  13 x 128 ->   26 x  26 x 128
>   20 route  19 8                                   ->   26 x  26 x 384
>   21 conv    256       3 x 3/ 1     26 x  26 x 384 ->   26 x  26 x 256 1.196 BF
>   22 conv     18       1 x 1/ 1     26 x  26 x 256 ->   26 x  26 x  18 0.006 BF
>   23 yolo
> [yolo] params: iou loss: mse (2), iou_norm: 0.75, cls_norm: 1.00, scale_x_y: 1.00
> Total BFLOPS 5.448
>  Allocate additional workspace_size = 12.46 MB
> Loading weights from backup\plate-yolov3-tiny_obj_last.weights...
>  seen 64
> Done! Loaded 24 layers from weights-file
> data/test.jpg: Predicted in 44.127000 milli-seconds.
```

**Comment by ogulcankarayel5** [comment]
Also this is my cfg file 
[net]
# Testing
#batch=64
#subdivisions=64
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear



[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=0
